### PR TITLE
working radius function

### PR DESCRIPTION
**Pull request by mbychkowski**
Radius drop down added below title. Search updates upon new selection. User has option of 5, 10, 25, 50 mile search

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -103,6 +102,19 @@ function searchFourSquare(search) {
   }); // Completes the function that pulls down the response
 } // Completes the entire function that searches foursquare
 
+// Initialize search
+function initSearch() {
+  searchBar = true;
```
Comment (mjsimon91) [inline script.js:107]: Is this still necessary? Trying to find where else this is used and not seeing it now

File: style.css [hunk 2]
```diff
@@ -3,12 +3,9 @@
   width: 100%;
 }
 
-.searchBox {
-  color: white;
-}
-
 #search {
-  padding-left: 10px;
```
Comment (mjsimon91) [inline style.css:11]: Why did you remove this? The padding allowed the text to not be right against the search box border

**Review by mjsimon91 — APPROVED** [review]
Looks goot. I like moving the initialization into its own function. Makes the code much cleaner. Please add the padding-left back to the search box as it pushed the text away from the border

File: style.css (hunk 2 above)
Comment (mbychkowski) [reply style.css:11]: Sorry about this, I noticed I did something to make it right against the search, but I did not know what!

File: script.js (hunk 1 above)
Comment (mbychkowski) [reply script.js:107]: I don't see what this comment is referring too
